ttnn-visualizer/src/components/GoldenTensorComparisonIndicator/styles.scss

```diff
@@ -0,0 +1,5 @@
+.performance-square {
```

Comment:
if the scss file is not scoped to only the component we shoudl consider namespacing to not pollute

Reply:
Wrote a more sensible class name.